irohad/ametsuchi/impl/temporary_wsv_impl.cpp

```diff
@@ -38,13 +38,14 @@ namespace iroha {
     }
 
     bool TemporaryWsvImpl::apply(
-        const model::Transaction &transaction,
-        std::function<bool(const model::Transaction &, WsvQuery &)>
-            apply_function) {
-      const auto &tx_creator = transaction.creator_account_id;
+        const shared_model::interface::Transaction &tx,
+        std::function<bool(const shared_model::interface::Transaction &,
+                           WsvQuery &)> apply_function) {
+      auto transaction = std::unique_ptr<model::Transaction>(tx.makeOldModel());
```

Comment:
Why don't use shared_model transaction instead? Looks like `transaction` by itself doesn't use.

Reply:
We don't have executors for new transactions

Reply:
But model::transaction by itself doesn't using in the code.